.changeset/perfect-kangaroos-tan.md

```diff
@@ -0,0 +1,7 @@
+---
+'@journeyapps/react-native-quick-sqlite': minor
+---
+
+- Remove use of nativeCallSyncHook() for new architecture support.
+- Bump react native version
```

Comment:
Are these React Native and Ruby versions relevant to consumers of this package? `react-native` seems to be a dev dependency here, I'm not sure how the Ruby version is used.

Reply:
I've removed those messages as it isn't relevant to users of the package.